docs/iot/guides/update-control-api.md

```diff
@@ -0,0 +1,134 @@
+# Tizen Update Control Native API
+
+- [Prerequisites](#prerequisites)
```

Comment:
You don't need to create this kind of list for the titles in MD file.
So, please remove line 3~ line 8.